src/components/member/tabs/contracts-tab/index.tsx

```diff
@@ -45,6 +51,10 @@ export const ContractTab: React.FC<{
           }
         />
       ))}
+      {trials.length > 0 ? <MainHeadline>Trials</MainHeadline> : null}
+      {trials.map((t) => (
+        <TrialComponent key={t.id} trial={t} />
+      ))}
```

Comment:
No 1 letter variable names please 🙏 
```suggestion
      {trials.map((trial) => (
        <TrialComponent key={trial.id} trial={trial} />
      ))}
```